controllers/admin/AdminPerformanceController.php

```diff
@@ -746,7 +746,7 @@ public function postProcess()
 			{
 				$theme_cache_directory = _PS_ALL_THEMES_DIR_.$this->context->shop->theme_directory.'/cache/';
 				if (((bool)Tools::getValue('PS_CSS_THEME_CACHE') || (bool)Tools::getValue('PS_JS_THEME_CACHE')) && !is_writable($theme_cache_directory))
-					$this->errors[] = Tools::displayError(sprintf($this->l('To use Smart Cache directory %s must be writable.'), realpath($theme_cache_directory)));
+					$this->errors[] = sprintf(Tools::displayError($this->l('To use Smart Cache directory %s must be writable.')), realpath($theme_cache_directory));
```

Comment:
Hi, I think $this->l() is useless here, because the string is translated directly into Tools::displayError().
Do you think so ?

Best regards.